### ISSUE TITLE
TBB: a solution to thread oversubscription?

### ISSUE THREAD

**Issue by chris-monardo** [flame/blis#656]
Hello, I'm looking to use BLIS but I have a concern about thread oversubscribing due to non-composability of OpenMP. I am developing a highly parallel program using TBB flow graphs where each thread may in turn use parallel BLIS operations. I'd like to request having a configure option to support TBB to solve the oversubscription problem.

For more information see Chapter 9 of ProTBB **Putting It All Together**. https://link.springer.com/book/10.1007/978-1-4842-4398-5

**Comment by jeffhammond** [comment]
I thought https://github.com/devinamatthews/tblis supported TBB but I can't tell anymore. @devinmatthews knows.

**Comment by devinamatthews** [comment]
Yes, TBLIS has TBB support and it is almost as performant as optimized OpenMP. The idea is to port this to BLIS at some point but there is no ETA yet.
